src/components/CreateReview/index.js

```diff
@@ -16,9 +16,10 @@ const CreateReview = ({movieId, refresh}) => {
         reviewServices.createReview(MY, review)
             .then((res) => refresh())
             .catch(errorServices.alertError);
+        setReview({...review, review: ""});
```

Comment:
I believe you're setting the review state to be the default? Then it's better you directly pass in the initial state, i.e. ```{movieId: movieId, rating: "0", review: ""}```. 

Now the review text is reset, but the rating is still the previous value.

Reply:
I think they are the same thing, unless movieid or rating changes in the mean time. But, I have made the change.